docs/getting_started/tutorial.md

```diff
@@ -0,0 +1,867 @@
+# Your first Wagtail site
+
+```eval_rst
+.. note::
+   This tutorial covers setting up a brand new Wagtail project.
+   If you'd like to add Wagtail to an existing Django project instead, see :doc:`integrating_into_django`.
+```
+
+## Install and run Wagtail
+
+### Install dependencies
+
+Wagtail supports Python 3.6, 3.7, 3.8 and 3.9.
+
+To check whether you have an appropriate version of Python 3:
+
+```sh
```

Comment:
Doesn't seem to be any syntax highlighting coming through here, should this be ``bash``?

Reply:
I just tried, neither is showing any particular highlighting on the line. 

... Ooh, it's actually still `console` that works. I had never seen that before.

Reply:
Fixed in a4551e996aba65c960316c30d532f034da03ed7e